protobuf/protoc-gen-govalidator/plugin/plugin.go

```diff
@@ -644,6 +644,13 @@ func genStringValidator(g *protogen.GeneratedFile, f *protogen.Field, varName st
 		}
 	}
 
+	if rules.GetPrefix() != "" {
```

Comment:
How does this affect the length check? I think it gets a bit confusing potentially so we have to include that [in the docs](https://safetyculture.atlassian.net/wiki/spaces/SEC/pages/2916024516/String+Validator). The prefix is included in the length check currently I suppose. Is that intended?